docs/static/openapi.json

```
@@ -876,6 +876,11 @@
                     "type": "string",
                     "nullable": true
                   },
+                  "context_msg": {
+                    "type": "string",
+                    "nullable": true,
+                    "description": "Additional system instructions for the agent task"
```

```suggestion
                    "description": "Context for the agent task. These are optional instructions dependent on the entry point"
```
Maybe... Idk, feel free to not accept it if it doesn't sound right to you.

---

For an end facing user I think the part about entry points may be less obvious

Perhaps 

> Optional context provides additional information to help the agent address the initial user request